api/app/src/main/kotlin/packit/controllers/RunnerController.kt

```diff
@@ -4,64 +4,53 @@ import org.springframework.http.ResponseEntity
 import org.springframework.security.access.prepost.PreAuthorize
 import org.springframework.validation.annotation.Validated
 import org.springframework.web.bind.annotation.*
-import packit.model.dto.GitBranches
-import packit.model.dto.OrderlyRunnerVersion
-import packit.model.dto.Parameter
-import packit.model.dto.RunInfoDto
-import packit.model.dto.RunnerPacketGroup
-import packit.model.dto.SubmitRunInfo
+import packit.model.dto.*
 import packit.service.RunnerService
 
 @RestController
 @RequestMapping("/runner")
 @PreAuthorize("hasAuthority('packet.run')")
-class RunnerController(private val runnerService: RunnerService)
-{
+class RunnerController(private val runnerService: RunnerService) {
```

Comment:
What's with the formatting changes in this file - they're not consistent with the other modules in the project I don't think.. ?